example/windows/PDFExample/pch.h

```diff
@@ -0,0 +1,29 @@
+﻿#pragma once
+
+#define NOMINMAX
+
+#include <hstring.h>
+#include <restrictederrorinfo.h>
+#include <unknwn.h>
+#include <windows.h>
+#include <winrt/Windows.ApplicationModel.Activation.h>
+#include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+#include <winrt/Windows.UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.Data.h>
+#include <winrt/Windows.UI.Xaml.Interop.h>
+#include <winrt/Windows.UI.Xaml.Markup.h>
+#include <winrt/Windows.UI.Xaml.Navigation.h>
+#include <winrt/Windows.UI.Xaml.h>
+
+#include <winrt/Microsoft.ReactNative.h>
+
+#include <winrt/Microsoft.UI.Xaml.Automation.Peers.h>
+#include <winrt/Microsoft.UI.Xaml.Controls.Primitives.h>
+#include <winrt/Microsoft.UI.Xaml.Controls.h>
+#include <winrt/Microsoft.UI.Xaml.Media.h>
+#include <winrt/Microsoft.UI.Xaml.XamlTypeInfo.h>
+
+#include "winrt/RCTPdf.h"
+#include "winrt/progress_view.h"
```

Comment:
This is different from the docs, which specify "progress-view" instead of "progress_view"

Reply:
Thank you for the heads-up. Will submit a PR to fix this.

Reply:
Looks like there's PR open already: https://github.com/wonday/react-native-pdf/pull/532